lib/routes.js

```diff
@@ -291,6 +291,9 @@ const METHOD_MAP = {
   '/wd/hub/session/:sessionId/appium/device/shake': {
     POST: {command: 'mobileShake'}
   },
+  '/wd/hub/session/:sessionId/appium/device/system_time': {
+    GET: {command: 'getDeviceDateAndTime'}
```

Comment:
do you think that 'time' implies 'date' here? if so I'd prefer the less verbose 'getDeviceTime'